### PR TITLE
fix: use correct FOSSA badge in README

### PR DESCRIPTION
**Pull request by ndrscodes**
uses the correct FOSSA badge for the vue-svg-loader-2 repository

### CODE REVIEW

**Comment by ndrscodes** [comment]
License checks are failing because of the legacy /docs folder. We'll put this on hold until #41 is merged.

**Comment by ndrscodes** [comment]
@kwilfert 
@Bachenbenno 
The license check is showing a wrong result. It failed due to the legacy `/docs` folder, which is not present anymore in the current build. I could manually rebase however, should you want proof ;)

**Comment by kwilfert** [comment]
Since the whole FOSSA thing is new to me, I'd kindly request for you to rebase it, if you really don't mind :)

**Review by kwilfert — APPROVED** [review]
Thanks for rebasing. I am now convinced the FOSSA Check works and is correctly triggered when changes are pushed.

**Pull request merged**
